crates/diesel_models/src/payment_attempt.rs

```diff
@@ -565,6 +573,22 @@ impl From<PaymentAttemptUpdate> for PaymentAttemptUpdateInternal {
                 updated_by,
                 ..Default::default()
             },
+            PaymentAttemptUpdate::BlocklistUpdate {
+                status,
+                connector,
+                error_code,
+                error_message,
+                updated_by,
+                merchant_connector_id,
+            } => Self {
+                status: Some(status),
+                error_code,
+                connector: Some(connector),
+                error_message,
+                updated_by,
+                merchant_connector_id: Some(merchant_connector_id),
```

Comment:
```suggestion
                connector: Some(None),
                error_message,
                updated_by,
                merchant_connector_id: Some(None),
```
You can hardcode this here since we want to ensure that the `connector` and `merchant_connector_id` fields are set to null in the DB.